### PR TITLE
Usage example for draw circle

### PR DESCRIPTION
**Pull request by daosgava**
# Overview

A usage example has been added for the drawCircle function, implemented in C++, Python, C# (top-level code), and C# (OOP).

Splashkit Function: [Draw Circle](https://splashkit.io/api/graphics/#draw-circle-1)

# Files Included
- [x] Title and explanation (.txt)
- [x] C++ code
- [x] C# code (top-level statements)
- [x] C# code (Object-Oriented Programming)
- [x] Python code
- [x] Screenshot

# Usage Example Checks
- [x] Simple, clear demonstration of the function
- [x] Code uses Splashkit functions
- [x] Tested in Chrome and Firefox

### CODE REVIEW

File: public/usage-examples/graphics/draw_circle_record/draw_circle_record-1-simple-oop.cs [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+            DrawCircle(ColorGreen(), circle4);
+
+            RefreshScreen();
+
+            Delay(4000);
+
+            CloseAllWindows();
+        }
+    }
+}
```
Comment (breezy-codes) [inline public/usage-examples/graphics/draw_circle_record/draw_circle_record-1-simple-oop.cs:33]: C# OOP code cannot use - `using static SplashKitSDK.SplashKit;`

Please refer to the documentation site on how to do C# OOP program codes and then fix this program.
https://thoth-tech.netlify.app/products/splashkit/splashkit-website/tutorials-documentation/04-oop-styling/

File: public/usage-examples/graphics/draw_circle_record/draw_circle_record-1-simple-oop.cs [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+            // Define the center point for all circles
+            Point2D center = new Point2D { X = 400, Y = 300 };
+
+            // Create four circles with different radius
+            Circle circle1 = new Circle { Center = center, Radius = 50 };
+            Circle circle2 = new Circle { Center = center, Radius = 100 };
+            Circle circle3 = new Circle { Center = center, Radius = 150 };
+            Circle circle4 = new Circle { Center = center, Radius = 200 };
+
+            // Draw the circles with different colors
```
Comment (breezy-codes) [inline public/usage-examples/graphics/draw_circle_record/draw_circle_record-1-simple-oop.cs:20]: This comment is different over all 4 files, please update so its the exact same comment on all.

File: public/usage-examples/graphics/draw_circle_record/draw_circle_record-1-simple.cpp [hunk 3]
```diff
@@ -0,0 +1,31 @@
+#include "splashkit.h"
+
+int main()
+{
+    open_window("Draw Circles", 800, 600);
+
+    clear_screen();
+
+    // Define the center of the circles
```
Comment (breezy-codes) [inline public/usage-examples/graphics/draw_circle_record/draw_circle_record-1-simple.cpp:9]: This comment line is different, in both C# files you do -
`// Define the center point for all circles`

But in python and cpp this is a different comment. Please update to ensure the comment is the same over all 4 files.

File: public/usage-examples/graphics/draw_circle_record/draw_circle_record-1-simple.cpp [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    point_2d center = {400, 300};
+
+    // Create 4 circles with different radii
```
Comment (breezy-codes) [inline public/usage-examples/graphics/draw_circle_record/draw_circle_record-1-simple.cpp:12]: Same issue with this comment.

**Review by breezy-codes — CHANGES_REQUESTED** [review]
I've left comments on some sections of your code on minor fixes and changes. 

Please update the description of this pull request to match the required one for usage examples specified here - 
https://thoth-tech.netlify.app/products/splashkit/splashkit-website/usage-examples/03-usage-pull-request/

Then make the requested changes.

**Review by breezy-codes — APPROVED** [review]
All requested changes have been made, code runs correctly and meets the standards of how usage examples should be for the site.
I approve this PR

- [X] All required files are present.
  - [X] Title and explanation (.txt)
  - [X] C++ code
  - [X] C# code (top-level statements)
  - [X] C# code (Object-Oriented Programming)
  - [X] Python code
- [X] Code correctly uses Splashkit functions.
- [X] Code clearly demonstrates the function.
- [X] All versions maintain the same structure and comments.
- [X] C++ code ran correctly.
- [X] C# top level code ran correctly.
- [X] C# OOP code ran correctly.
- [X] Python code ran correctly.

**Review by huyhuynh2319 — APPROVED** [review]
I've reviewed the usage examples. Everything ran without issue.

### **Checks**

- [x]  All required files are present.
- [x]  Title and explanation (.txt)
- [x]  C++ code
- [x]  C# code (top-level statements)
- [x]  C# code (Object-Oriented Programming)
- [x]  Python code

### **Tests done**

- [x]  C++ code ran correctly.
- [x]  C# top level code ran correctly.
- [x]  C# OOP code ran correctly.
- [x]  Python code ran correctly.

**Review by omckeon — APPROVED** [review]
I've made some changes to the code, just to make this clearer, and to ensure no OO code in the top level C# code.

**Pull request merged**